src/index.js

```diff
@@ -0,0 +1,5 @@
+let button=document.querySelector(".mybutton") 
+
+button.addEventListener("click",()=>{
```

Comment:
you should add a button tag to your HTML in order for this to work properly (gives me an error when I open it whit live server)